_extras/snippets_library/UCL_sge/16/bench.snip

```diff
@@ -0,0 +1,239 @@
+```
+
+Submit the job with `{{ site.sched_submit }}`:
+
+```
+{{ site.host_prompt }} {{ site.sched_submit }} calc_pi.sh
+```
+{: .bash}
+
+```
+Your job 4053574 ("Serial_Job") has been submitted
+```
+{: .output}
+
+```
+{{ site.host_prompt }} {{ site.sched_status }}
+```
+{: .bash}
+
+```
+job-ID  prior   name       user         state submit/start at     queue                          slots ja-task-ID
+-----------------------------------------------------------------------------------------------------------------
+4053574 0.00000 Serial_Job cceagil      qw    07/02/2020 15:34:40
+```
+{: .output}
+
+By periodically checking the queue with `{{ site.sched_status }}`, we can see when our job is running:
+
+```
+{{ site.host_prompt }} {{ site.sched_status }}
+```
+{: .bash}
+
+```
+job-ID  prior   name       user         state submit/start at     queue                          slots ja-task-ID
+-----------------------------------------------------------------------------------------------------------------
+4053574 3.50000 Serial_Job cceagil      r     07/02/2020 15:35:53 Arya@node-f00a-001                 1
+```
+{: .output}
+
+When the job is running, basic information about it can be obtained with `nodesforjob`:
+
+```
+{{ site.host_prompt }} nodesforjob 4053574
+```
+{: .bash}
+
+```
+Nodes for job 4053574:
+  Primary:
+    node-f00a-001:  21.9 % load, 22.3 % memory used, 1.1% swap used
+```
+{: .output}
+
+
+
+`{{ site.sched_hist }}`` gives more information:
+
+```
+{{ site.host_prompt }} {{ site.sched_hist }}
+```
+{: .bash}
+
+```
+        FSTIME        |       FETIME        |   HOSTNAME    |  OWNER  | JOB NUMBER | TASK NUMBER | EXIT STATUS |  JOB NAME   
+----------------------+---------------------+---------------+---------+------------+-------------+-------------+-------------
+  1970-01-01 01:00:00 | 1970-01-01 01:00:00 | node-i00a-002 | cceagil |    4029963 |           0 |           0 | Serial_Job  
+  2020-07-01 16:12:31 | 2020-07-01 16:12:31 | node-b00a-007 | cceagil |    4030977 |           0 |           0 | Serial_Job  
+  2020-07-01 16:38:40 | 2020-07-01 16:38:42 | node-i00a-002 | cceagil |    4031101 |           0 |           0 | Serial_Job  
+  2020-07-02 15:37:56 | 2020-07-02 15:37:58 | node-f00a-001 | cceagil |    4053574 |           0 |           0 | Serial_Job
+```
+{: .output}
+
+Once the job has finished, if you now `cd` up one level, you should see files
+with names like `Serial_Job.e4053574` and `Serial_Job.o4053574`. These contain
+information about your job (e.g. the timing information) whose number was the one
+in the file title.
+
+If you go up another couple of levels, to your `Scratch` directory, you should find
+a compressed archive file with a name like `files_from_job_4053574.tar.gz`. The number in the file
+title should be the same as the `Serial.Job` files we looked at a moment ago.
+
+Extract the contents of the archive:
+
+```
+{{ site.host_prompt }} tar xvfz files_from_job_4053574.tar.gz
```

Comment:
```suggestion
{{ site.host_prompt }} tar zxvf files_from_job_4053574.tar.gz
```
since the filename is the argument for `f`, I'd prefer to see the f next to the filename